Prombench failing due to yarn dependency missing from Prometheus builder

Prombench is not able to start up the Prometheus instances for testing pull requests due to the yarn dependency missing. Logs:
```
2019-10-28T17:53:26.058914493Z >> Creating prometheus binaries
2019-10-28T17:53:26.173078238Z cd web/ui/react-app && yarn --frozen-lockfile
2019-10-28T17:53:26.173809471Z /bin/sh: 1: yarn: not found
2019-10-28T17:53:26.173922738Z Makefile:35: recipe for target 'web/ui/react-app/node_modules' failed
```
cc: @krasi-georgiev and @geekodour 
also @juliusv

--- a/temp/prometheus-builder/Dockerfile
+++ b/temp/prometheus-builder/Dockerfile
@@ -8,7 +8,11 @@ RUN \
         ca-certificates \
         make \
         git \
-	&& rm -rf /var/lib/apt/lists/*
+       && curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
+       && echo "deb https://dl.yarnpkg.com/debian/ stable main" > /etc/apt/sources.list.d/yarn.list \
+       && apt-get update \
+       && apt-get install -y --no-install-recommends nodejs yarn \
+       && rm -rf /var/lib/apt/lists/*
 
 RUN mkdir -p /go/src/github.com
 COPY build.sh /go/src/github.com/build.sh

